scss/components/_case-list-table.scss

```diff
@@ -0,0 +1,139 @@
+.civicase__table--caselist {
+  .fixed-col {
```

Comment:
What is the usage of `fixed-col`? 
and all class names needs to be converted to BEM.

Reply:
Done